docs/src/main/asciidoc/datastore.adoc

```diff
@@ -1191,6 +1191,8 @@ Spring Data Cloud Datastore publishes events extending the Spring Framework's `A
 [options="header"]
 |===
 | Type | Description | Contents
+| `AfterKeyReadEvent`|Published immediately after read by-key operations are executed by `DatastoreTemplate`| The entities read from Cloud Datastore and the original keys in the request.
+| `AfterQueryReadEvent`|Published immediately after read byquery operations are executed by `DatastoreTemplate`| The entities read from Cloud Datastore and the original query in the request.
```

Comment:
Here also, I would consider calling it `AfterReadByQueryEvent` or just simply `AfterQueryEvent`.

Reply:
Yea can do